conf/application.conf

```diff
@@ -28,12 +28,12 @@ play.http.filters = play.api.http.EnabledFilters
 play.filters.enabled = [
   play.filters.headers.SecurityHeadersFilter,
   com.kenshoo.play.metrics.MetricsFilter,
-  uk.gov.hmrc.play.bootstrap.filters.frontend.HeadersFilter,
-  uk.gov.hmrc.play.bootstrap.filters.frontend.crypto.SessionCookieCryptoFilter,
-  uk.gov.hmrc.play.bootstrap.filters.frontend.deviceid.DeviceIdFilter,
+  uk.gov.hmrc.play.bootstrap.frontend.filters.HeadersFilter,
+  uk.gov.hmrc.play.bootstrap.frontend.filters.crypto.SessionCookieCryptoFilter,
```

Comment:
@carrielies I have just discovered that we can import `play.http.filters = "uk.gov.hmrc.play.bootstrap.frontend.filters.FrontendFilters"` rather than needing to specify them all in the array.

Reply:
Nice, I'll make that change